mlflow/protos/service.proto

```diff
@@ -204,6 +204,23 @@ service MlflowService {
     };
   }
 
+  // Log a metric for a run in async fashion. A metric is a key-value pair (string key, float value) with an
+  // associated timestamp. Examples include the various metrics that represent ML model accuracy.
+  // A metric can be logged multiple times.
+  //
+  rpc logMetricAsync(LogMetricAsync) returns (LogMetricAsync.Response) {
```

Comment:
I think this proto can come as a next step in the OSS repo (unless everyone wants one already) - you can modify our internal protos for the service

Reply:
100% agree. Would prefer not to extend protos / REST API for now :). cc @BenWilson2 @prithvikannan @yunpark93